### DISCUSSION TITLE
Statistic 统计数值 控件， 可以加一个 说明属性

### DISCUSSION THREAD

**Discussion by ekuku1984** [ant-design/ant-design#51740] [Ideas]
大部分情况下， 会对统计值加一个说明， 现在不方便加， 虽然可以通过 valueRender 来解决一些问题， 但是有 prefix, suffix 时， valueRender 就不适合了。

可以加个 extra 或 description 等属性

**Comment by petercat-assistant** [comment]
您提到在 `Statistic` 组件中希望增加一个说明属性，以便在有 `prefix` 和 `suffix` 的情况下更方便地添加说明。这是一个很好的建议，可以提升组件的灵活性。

目前，`Statistic` 组件确实可以通过 `valueRender` 来自定义显示内容，但在同时使用 `prefix` 和 `suffix` 时可能会遇到一些限制。增加一个 `extra` 或 `description` 属性可以更好地解决这个问题。

如果您希望推动这个功能的实现，建议您在 ant-design 的 GitHub 仓库中创建一个功能请求（Feature Request）。这样，开发团队可以评估这个建议，并考虑在未来的版本中进行实现。

如果您有其他问题或需要进一步的帮助，请在评论中描述您的问题并 @petercat-assistant 以开始与我对话。
